test/markdown-lint-config.rc

```diff
@@ -0,0 +1,5 @@
+# For help, see
+# https://github.com/markdownlint/markdownlint/blob/master/docs/configuration.md 
+
+# Ignore long lines
```

Comment:
MD013 is about long lines. I suggest briefly describing what other rules are being ignored, so the next person updating this file doesn't need hunt them down in the documentation.

Reply:
@adrcunha good point :+1: